mesh/nurbs.cpp

```diff
@@ -2629,7 +2877,8 @@ void NURBSExtension::LoadSolution(std::istream &input, GridFunction &sol) const
          {
             for (int i = 0; i < nx; i++)
             {
-               const int l = (kv.Size() == 2) ? p2g(i,j) : p2g(i,j,k);
+               const int ll = (kv.Size() == 2) ? p2g(i,j) : p2g(i,j,k);
+               const int l  = DofMap(l);
```

Comment:
The above two lines generate warnings: `ll` is unused and `l` is used in its own initialization.